app/Main.hs

```diff
@@ -1,7 +1,11 @@
 module Main (main) where
 
-import Hintman (someFunc)
+import Network.Wai.Handler.Warp   (run)
```

Comment:
I've put the `.stylish-haskell.yaml` file into this project. You can run `stylish-haskell` to not have any issues with the imports alignment.